tests/test_workflows/test_rolling_workflow.py

```diff
@@ -0,0 +1,36 @@
+import pytest
+import pandas as pd
+from pathlib import Path
+from waveanalysis.data_workflows import rolling_workflow
+
+@pytest.fixture
+def default_log_params():
+    return {
+        'Box Size(px)': 20,
+        'Box Shift(px)': 20,
+        'Base Directory': 'tests/assets/rolling',
+        'ACF Peak Prominence': 0.1,
+        'Files Processed': [],
+        'Files Not Processed': [],
+        'Plotting errors': [],
+        'Submovies Used' : [],
+        'Errors': [],
+        'Frame Interval': [],
+        'Pixel Size': []
+        }
+
+
+def test_rolling_workflow(default_log_params):
+    # load csv
+    known_results = pd.read_csv('tests/assets/rolling/1_Group2_summary.csv')
```

Comment:
it looks like these `.csv` files didn't get committed to the repo, so these tests fail

Reply:
Probably because .csv is in the .gitignore file, oops! you can use git add -f path/to/your/file.csv to force add a .csv file